content/changelog/2024-07-12-function-signatures.md

```diff
@@ -1,6 +1,6 @@
 ---
 author: ["Ned Batchelder"]
```

Comment:
Does it make sense to have individuals listed as authors for changelog snippets, if the board acts with collective responsibility? Perhaps remove the author, or use "the board"?

Reply:
I'm seeing author as "who wrote up this whole post" as opposed to "who made the decision written in this post", but not sure if the rest of the board feels differently.